### PR TITLE
PascalCase property names

### PR DESCRIPTION
**Pull request by meJevin**
Thanks @dgvives!

### CODE REVIEW

**Comment by meJevin** [comment]
Leaked my test token again 🙄

File: Source/Cryptlex.Net/Core/Services/AccountsService.cs [hunk 1]
```diff
@@ -17,9 +17,9 @@ public interface IAccountsService :
     {
         Task<AccountLoginResponse> Login(AccountLoginData data);
         Task<AccountLoginGoogleResponse> LoginGoogle(AccountLoginGoogleData data);
-        Task<bool> CheckSSOEnabled(string companyId);
-        Task<Uri> InitiateSSOLogin(string companyId, string returnUrl);
-        Task<Uri> GetSSOReturnUrl(string companyId);
+        Task<bool> CheckSSOEnabled(string CompanyId);
+        Task<Uri> InitiateSSOLogin(string CompanyId, string ReturnUrl);
```
Comment (dgvives) [inline Source/Cryptlex.Net/Core/Services/AccountsService.cs:21]: As per Microsoft C# coding conventions https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions 
Property names are PascalCase, while parameters and internal variables are camelCase

```csharp
Task<Uri> InitiateSSOLogin(string companyId, string returnUrl);
```

File: Source/Cryptlex.Net/Entities/Account.cs [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-        public string? googleClientId { get; set; }
-        public DateTime trialExpiresAt { get; set; }
-        public Plan plan { get; set; }
-        public string? companyId { get; set; }
+        public string? Id { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public DateTime UpdatedAt { get; set; }
+        public string? Company { get; set; }
+        public string? Email { get; set; }
+        /// <summary>
```
Comment (dgvives) [inline Source/Cryptlex.Net/Entities/Account.cs:10]: 👍

File: Source/Cryptlex.Net/Entities/Account.cs [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
+        /// "active" "inactive"
+        /// </summary>
+        public string? Status { get; set; }
```
Comment (dgvives) [inline Source/Cryptlex.Net/Entities/Account.cs:13]: 👍

File: Source/Cryptlex.Net/Requests/Accounts/AccountCheckSSOEnabledData.cs [hunk 4]
```diff
@@ -1,12 +1,14 @@
-﻿namespace Cryptlex.Net.Accounts
+using System.Text.Json.Serialization;
+namespace Cryptlex.Net.Accounts
 {
     public class AccountCheckSSOEnabledData
     {
-        public string comapnyId { get; set; }
+		[JsonPropertyName("comapnyId")]
+        public string ComapnyId { get; set; }
```
Comment (dgvives) [inline Source/Cryptlex.Net/Requests/Accounts/AccountCheckSSOEnabledData.cs:7]: companyId has been renamed to accountAlias on the last Cryptlex WebAPI update 

https://api.cryptlex.com/v3/docs#tag/Authentication/operation/post/v3/accounts/login/saml/verify

File: Source/Cryptlex.Net/Entities/Activation.cs [hunk 5]
```diff
@@ -1,27 +1,30 @@
...
-        public List<ActivationMetadata>? metadata { get; set; }
-        public string? licenseId { get; set; }
-        public DateTime? lastSyncedAt { get; set; }
-        public DateTime? leaseExpiresAt { get; set; }
-        public List<ActivationMeterAttribute>? meterAttributes { get; set; }
+        public string? Id { get; set; }
+        public DateTime? CreatedAt { get; set; }
+        public DateTime? UpdatedAt { get; set; }
+        /// <summary>
+        /// "windows" "linux" "macOs" "android" "iOs"
```
Comment (dgvives) [inline Source/Cryptlex.Net/Entities/Activation.cs:12]: It is lowercase
https://api.cryptlex.com/v3/docs#tag/Activations/operation/get/v3/activations/%7Bid%7D

```
"windows" "linux" "macos" "android" "ios"
```

File: Source/Cryptlex.Net/Entities/TrialActivation.cs [hunk 6]
```diff
@@ -8,19 +8,22 @@ namespace Cryptlex.Net.Entities
...
-        public string? productId { get; set; }
-        public DateTime? expiresAt { get; set; }
-        public List<ActivationMetadata>? metadata { get; set; }
+        public string? Id { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public DateTime UpdatedAt { get; set; }
+        /// <summary>
+        /// "windows" "linux" "macOs" "android" "iOs"
+        /// </summary>
+        public string? Os { get; set; }
```
Comment (dgvives) [inline Source/Cryptlex.Net/Entities/TrialActivation.cs:17]: Same here, possible values are lowercase
https://api.cryptlex.com/v3/docs#tag/Activations/operation/get/v3/activations/%7Bid%7D

```
"windows" "linux" "macos" "android" "ios"
```

File: Source/Cryptlex.Net/Requests/Activations/CreateActivationData.cs [hunk 7]
```diff
@@ -9,35 +10,50 @@ namespace Cryptlex.Net.Activations
...
-        public string userHash { get; set; }
-        public string productId { get; set; }
-        public string key { get; set; }
-        public string? email { get; set; }
-        public string? password { get; set; }
-        public int? leaseDuration { get; set; }
-        public List<ActivationMetadataRequestModel>? metadata { get; set; }
-        public List<ActivationMeterAttributeRequestModel>? meterAttributes { get; set; }
+		[JsonPropertyName("os")]
+        public string Os { get; set; }
```
Comment (dgvives) [inline Source/Cryptlex.Net/Requests/Activations/CreateActivationData.cs:14]:
```
/// <summary>
/// Name of the operating sistem
/// Enum: "windows" "linux" "macos" "android" "ios" 
/// </summary>
```

File: Source/Cryptlex.Net/Requests/Activations/UpdateActivationData.cs [hunk 8]
```diff
@@ -9,34 +10,49 @@ namespace Cryptlex.Net.Activations
...
-        public string userHash { get; set; }
-        public string productId { get; set; }
-        public string key { get; set; }
-        public string? email { get; set; }
-        public string? password { get; set; }
-        public int? leaseDuration { get; set; }
-        public List<ActivationMetadataRequestModel>? metadata { get; set; }
-        public List<ActivationMeterAttributeRequestModel>? meterAttributes { get; set; }
+		[JsonPropertyName("os")]
+        public string Os { get; set; }
```
Comment (dgvives) [inline Source/Cryptlex.Net/Requests/Activations/UpdateActivationData.cs:14]:
```
/// <summary>
///  Name of the operating system
///  Enum: "windows" "linux" "macos" "android" "ios" 
/// </summary>
```

File: Source/Cryptlex.Net/Requests/Releases/CreateReleaseData.cs [hunk 9]
```diff
@@ -1,25 +1,34 @@
...
+		[JsonPropertyName("name")]
+        public string Name { get; set; }
+		[JsonPropertyName("version")]
+        public string Version { get; set; }
+		[JsonPropertyName("channel")]
+        public string Channel { get; set; }
+		[JsonPropertyName("platform")]
+        public string Platform { get; set; }
+		[JsonPropertyName("@private")]
         public bool? @private { get; set; }
```
Comment (dgvives) [inline Source/Cryptlex.Net/Requests/Releases/CreateReleaseData.cs:15]: Property name serialized should be string literal "private"
https://api.cryptlex.com/v3/docs#tag/Releases/operation/post/v3/releases

```
[JsonPropertyName("private")]
public bool? Private { get;  set; }
```

File: Source/Cryptlex.Net/Requests/Releases/UpdateReleaseData.cs [hunk 10]
```diff
@@ -1,14 +1,23 @@
...
+		[JsonPropertyName("name")]
+        public string? Name { get; set; }
+		[JsonPropertyName("version")]
+        public string? Version { get; set; }
+		[JsonPropertyName("channel")]
+        public string? Channel { get; set; }
+		[JsonPropertyName("platform")]
+        public string? Platform { get; set; }
+		[JsonPropertyName("@private")]
         public bool? @private { get; set; }
```
Comment (dgvives) [inline Source/Cryptlex.Net/Requests/Releases/UpdateReleaseData.cs:15]:
```
[JsonPropertyName("private")]
public bool? Private { get; set; }
```

File: Source/Cryptlex.Net/Requests/EmailTemplates/CreateEmailTemplateData.cs [hunk 11]
```diff
@@ -1,32 +1,44 @@
...
+        public string? Cc { get; set; }
+		[JsonPropertyName("bcc")]
+        public string? Bcc { get; set; }
+		[JsonPropertyName("subject")]
+        public string Subject { get; set; }
+		[JsonPropertyName("body")]
+        public string Body { get; set; }
+		[JsonPropertyName("replyTo")]
+        public string? ReplyTo { get; set; }
+		[JsonPropertyName("@event")]
```
Comment (dgvives) [inline Source/Cryptlex.Net/Requests/EmailTemplates/CreateEmailTemplateData.cs:22]:
```csharp
[JsonPropertyName("event")]
public string Event{get;set;}
```

File: Source/Cryptlex.Net/Requests/EmailTemplates/UpdateEmailTemplateData.cs [hunk 12]
```diff
@@ -1,17 +1,29 @@
...
+        public string? Cc { get; set; }
+		[JsonPropertyName("bcc")]
+        public string? Bcc { get; set; }
+		[JsonPropertyName("subject")]
+        public string? Subject { get; set; }
+		[JsonPropertyName("body")]
+        public string? Body { get; set; }
+		[JsonPropertyName("replyTo")]
+        public string? ReplyTo { get; set; }
+		[JsonPropertyName("@event")]
```
Comment (dgvives) [inline Source/Cryptlex.Net/Requests/EmailTemplates/UpdateEmailTemplateData.cs:22]:
```csharp
[JsonPropertyName("event")
public string? Event{get;set;}
```

File: Source/Cryptlex.Net/Requests/Accounts/AccountCheckSSOEnabledData.cs (hunk 4 above)
Comment (meJevin) [reply Source/Cryptlex.Net/Requests/Accounts/AccountCheckSSOEnabledData.cs:7]: Thanks! I'll get on it :)

**Pull request merged**